src/IO/VTK.jl

```diff
@@ -70,13 +76,39 @@ function save_vtk(
             for (name_i, array_i) in zip(data_names_v, data_arrays_v)
                 vtk[name_i] = Array(array_i)
             end
-            return vtk["Velocity"] = velocity_field
+            vtk["Velocity"] = velocity_field
```

Comment:
Perhaps it's better to make the time a keyword argument with `t=nothing` as default. Then here you can do 
```julia
isnothing(t) || (vtk["TimeValue"] = t)
```

Same in the function below.